packages/std/src/query/bank.rs

```diff
@@ -16,6 +20,14 @@ pub enum BankQuery {
     AllBalances { address: String },
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
```

Comment:
```suggestion
#[serde(rename_all = "snake_case")]
#[non_exhaustive]
```

Let's make this `#[non_exhaustive]` to ensure we can add query response fields later on in a non-breaking way.